### PR TITLE
Populate pama boxes

### PR DESCRIPTION
**Pull request by jmandel**

### CODE REVIEW

**Comment by jmandel** [comment]
image

**Comment by jmandel** [comment]
(As an add-on, I'm also making sure SMART web messaging can route back into these fields.)

File: src/components/Pama/pama.css [hunk 1]
```diff
@@ -8,10 +8,25 @@
...
+
+.current-selection {
+  padding-left: 4px;
+}
+
 .half-view {
   width: 50%;
 }
 
+div.choice {
```
Comment (dennispatterson) [inline src/components/Pama/pama.css]: We explicitly call out the other styles within pama.jsx.  Where does this come into play?

Comment (jmandel) [reply src/components/Pama/pama.css]: I think it's stray; will prune.

**Review by dennispatterson — APPROVED** [review]

**Pull request merged**